d2layouts/d2near/layout.go

```diff
@@ -98,7 +98,7 @@ func WithoutConstantNears(ctx context.Context, g *d2graph.Graph) (nears []*d2gra
 			nears = append(nears, obj)
 			g.Objects = append(g.Objects[:i], g.Objects[i+1:]...)
 			i--
-			delete(obj.Parent.Children, obj.ID)
+			delete(obj.Parent.Children, strings.ToLower(obj.ID))
```

Comment:
which test was this catching?

Reply:
`stable/constant_near_stress`

`len(obj.Children) = 5` and `len(other.Children) = 2` because `Joe`, `Donald`, `The ..` were not properly deleted with `d2near.WithoutConstantNears`

```
x -> y
The top of the mountain: { shape: text; near: top-center }
Joe: { shape: person; near: center-left }
Donald: { shape: person; near: center-right }
bottom: |md
	# Cats, no less liquid than their shadows, offer no angles to the wind.

  If we can't fix it, it ain't broke.

  Dieters live life in the fasting lane.
| { near: bottom-center }
i am top left: { shape: text; near: top-left }
i am top right: { shape: text; near: top-right }
i am bottom left: { shape: text; near: bottom-left }
i am bottom right: { shape: text; near: bottom-right }
```